sympy/stats/crv_types.py

```diff
@@ -1308,6 +1314,9 @@ def GammaInverse(name, a, b):
     >>> cdf(X)(z)
     Piecewise((uppergamma(a, b/z)/gamma(a), z > 0), (0, True))
 
+    >>> Y = GammaInverse("y",1,1)
+    >>> isinstance(sample(Y),float)
```

Comment:
Please use PEP-8 formatting while writing code in SymPy.
The line should be `Y = GammaInverse("y", 1, 1)`
Similarly, follow this for all other cases.